codewof/programming/content/en/evens-out/question.md

```diff
@@ -0,0 +1,7 @@
+# Evens Out
+
+The following function `evens_out(numbers)` is supposed to take a list of numbers and **return** a new list containing only the odd numbers from the original list.
+
+But there's a problem: it is leaving out all the *odd* numbers, instead of all the evens.
```

Comment:
I really like how this is worded and now regret how I made some of my debug questions. Good work :+1: